### PR TITLE
Increase Threshold for some tests where common failure occurs.

### PR DESCRIPTION
**Pull request by kartikdutt18**
Hey everyone,
This is an attempt to fix random failure in Azure due to a low threshold values. This PR aims to fix the same.
Thanks.

### CODE REVIEW

**Comment by Aakash-kaushik** [comment]
Hey @kartikdutt18 can you check out the mac os python build failure in the PR: #2621, I believe that is also a threshold failure and can be added to your PR.

**Comment by kartikdutt18** [comment]
Hey @Aakash-kaushik, following up on your comment, I have already fixed the threshold issue.
Also, I have re-run the tests.

File: src/mlpack/tests/feedforward_network_test.cpp [hunk 1]
```diff
@@ -148,10 +148,10 @@ TEST_CASE("CheckCopyMovingVanillaNetworkTest", "[FeedForwardNetworkTest]")
   model1->Add<Linear<> >(8, 3);
   model1->Add<LogSoftMax<> >();
 
-  // Check whether copy cpnstructor is working or not.
+  // Check whether copy constructor is working or not.
```
Comment (zoq) [inline src/mlpack/tests/feedforward_network_test.cpp:151]: Nice catch!

**Review by rcurtin — APPROVED** [review]
Hey @kartikdutt18, this looks great to me.  Thanks for taking the time to figure this out! :+1:

Do you want to fix the static code analysis issues before merge?  They should be pretty straightforward---it looks like everything in `ann_layer_test.cpp` is just suggesting the use of initialization lists, and the issue in `feedforward_network_test.cpp` could just be fixed by a line like `(void*) copiedModel;`.  Anyway, up to you---we could also open another issue for it if you want. :+1:

**Comment by kartikdutt18** [comment]
Hey @rcurtin, Sure, I'll push a commit for it.

**Comment by rcurtin** [comment]
Awesome, all the tests pass now.  Thanks @kartikdutt18! :+1:

**Comment by kartikdutt18** [comment]
Thanks a lot @rcurtin, @zoq for the review.

**Pull request merged**